docs/notebook/Tutorial.ipynb

```diff
@@ -521,15 +521,14 @@
     "\n",
     "OmegaConf support variable interpolation, Interpolations are evaluated lazily on access.\n",
     "\n",
-    "### Config node interpolation"
+    "## Config node interpolation"
```

Comment:
Please provide a screen shot, I am not good at reading json data.

Reply:
![image](https://user-images.githubusercontent.com/30913090/88211105-7d9dc700-cc55-11ea-91fb-48cb65077d45.png)